sdt_probe_offset.c

```diff
@@ -60,21 +60,24 @@ static long convert_addr_to_offset(Elf *elf_handle, size_t addr)
 	text_section_found = 0;
 
 	while((elf_section = elf_nextscn(elf_handle, elf_section)) != NULL) {
-		if (gelf_getshdr(elf_section, &elf_section_header) != &elf_section_header) {
-			fprintf(stderr, "GELF get section header failed: %s.\n", elf_errmsg(-1));
+		if (gelf_getshdr(elf_section, &elf_section_hdr) != &elf_section_hdr) {
```

Comment:
I think it would be clearer to have something like:
if (gelf_getshdr(elf_section, &elf_section_hdr) == NULL ) {
It seems more obvious that it's error checking since NULL pointers are often associated with errors.